### PR TITLE
Refactor, Cleaning and Tests

### PR DESCRIPTION
**Pull request by roscha80**
Added a test to my create page to test the form
small code cleaning/structure

### CODE REVIEW

File: client/src/components/SearchBar.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default function SearchBar({ onChange }) {
 
 const Wrapper = styled.div`
   display: grid;
+  padding: 10px 10px;
```
Comment (czintel) [inline client/src/components/SearchBar.js:27]: why not just "padding: 10px" ?

File: client/src/pages/HomePage.js [hunk 2]
```diff
@@ -10,16 +10,16 @@ export default function HomePage({ title, onThemeClick }) {
     <Wrapper>
       <Header>{title}</Header>
       <LogoWrapper>
-        <StyledLogo src={logo} alt="image" />
+        <LogoStyled src={logo} alt="image" />
 
         <H2Styled>Skills</H2Styled>
-        <StyledP>
+        <PStyled>
```
Comment (czintel) [inline client/src/pages/HomePage.js:16]: I'd go with StyledParagraph but that's a personal preference

**Review by czintel — APPROVED** [review]

**Comment by deriancastro** [comment]
Great work! It looks Ok for me, just a question: why Vercel again?

File: client/src/components/SearchBar.js (hunk 1 above)
Comment (roscha80) [reply client/src/components/SearchBar.js:27]: i only wanted padding on two sides :)

**Pull request merged**